iso690-numeric-brackets-cs.csl

```diff
@@ -35,9 +35,9 @@
   </macro>
   <macro name="editor">
     <names variable="editor">
-      <name and="text" name-as-sort-order="first" sort-separator=", " delimiter=", " delimiter-precedes-last="never">
-        <name-part name="given"/>
+      <name and="text" name-as-sort-order="all" sort-separator=", " delimiter=", " delimiter-precedes-last="never">
         <name-part name="family" text-case="uppercase"/>
+        <name-part name="given"/>
```

Comment:
The family name has to be upper case all, and the given name has to be upper case first letter, lower case rest. So if it stays this way (and now I see it probably will), yes it can be removed. Should I do it?
You know, I do not understand how the markup works to such details, so I just did minimal edits of the parent style. I did thought the order of these tags matters.
Thanks for your time.

Dne 14. 4. 2015 15:44 napsal uživatel Sebastian Karcher <notifications@github.com>:In iso690-numeric-brackets-cs.csl:
>          <name-part name="family" text-case="uppercase"/>
> +        <name-part name="given"/>

could we actually take all the <name-part name="give"/> out of this style? They don't actually do anything. cs:name-part is only used for styling (e.g. the uppercase used for family names here). It has no effect on either the presence of name parts or their order. 

—Reply to this email directly or view it on GitHub.